README.md

```diff
@@ -3,7 +3,7 @@
 [![Build Status](https://github.com/PrismJS/prism/workflows/CI/badge.svg)](https://github.com/PrismJS/prism/actions)
 [![npm](https://img.shields.io/npm/dw/prismjs.svg)](https://www.npmjs.com/package/prismjs)
 
-Prism is a lightweight, robust, elegant syntax highlighting library. It's a spin-off project from [Dabblet](https://dabblet.com/).
+Prism is a lightweight, robust and elegant syntax highlighting library. It's a spin-off project from [Dabblet](https://dabblet.com/).
```

Comment:
Pretty sure a comma is missing.

```suggestion
Prism is a lightweight, robust, and elegant syntax highlighting library. It's a spin-off project from [Dabblet](https://dabblet.com/).
```

Reply:
Sure thing I can revert those changes! As far as adding the oxford comma before the 'and'... I believe that it is optional to utilize. I'm happy to switch to using it, if you prefer.